Code/ww3d2/dllist.h

```diff
@@ -73,11 +73,10 @@ class DLListClass
 template <class T>
 class DLDestroyListClass : public DLListClass<T>
 {
-	using DLListClass<T>::Head;
 public:
 	virtual ~DLDestroyListClass()
 	{
-		while (T* t=Head()) {		
+		while (T* t=this->Head()) {		
```

Comment:
This seems out of scope for the purpose of the PR and should probably be a separate PR with its own discussion on the technical merits of `using` vs `this->`?